integrations/cypress/package.json

```diff
@@ -14,9 +14,11 @@
     "README.md"
   ],
   "dependencies": {
-    "@bigtest/globals": "^0.7.5",
-    "@bigtest/interactor": "^0.28.2",
-    "cypress": "^7.2.0"
+    "@bigtest/globals": "^0.7.6",
+    "@bigtest/interactor": "^0.29.0"
+  },
+  "peerDependencies": {
+    "cypress": "^8.0.0"
```

Comment:
Maybe we should make this `>= 8.0.0` so that it will be forward compatible?

Reply:
I think it won't make much of a difference whether it's `^` or `>=` (given how little tests I've written for it), but it would be nice to do a manual check for breaking changes before we tell people it's okay to use `9.0.0`. 🤷‍♂️ 

I think the bigger question might be if we should lower peerDependencies back down to `>= 6.0.0` now that we know it's not a recent change that introduced the cypress bug. So maybe something like `>=6.0.0 <9.0.0`

Reply:
We can also say `"^6.0.0,^7.0.0.,^8.0.0"`